Userland/Libraries/LibGUI/Widget.h

```diff
@@ -283,6 +289,11 @@ class Widget : public Core::Object {
     void set_accepts_emoji_input(bool b) { m_accepts_emoji_input = b; }
     bool accepts_emoji_input() const { return m_accepts_emoji_input; }
 
+    void set_context_menu_policy(GUI::ContextMenuPolicy policy) { m_context_menu_policy = policy; };
+    ContextMenuPolicy get_context_menu_policy() const { return m_context_menu_policy; }
```

Comment:
Getters do not have a `get_` prefix, this should be called `context_menu_policy()`.